vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -0,0 +1,57 @@
+# How to customize VizroAI
+
+This guide explains how to customize `VizroAI`.
+
+`VizroAI` accepts a single argument named `model`. This `model` argument can either be a string that specifies the name of the `ChatOpenAI` model or an instantiated `ChatOpenAI` model.
+
+We'll delve into customizing the `ChatOpenAI` model later; for now, let's concentrate on the default usage of `VizroAI`.
+
+## Default usage
+
+`VizroAI`can be initialized without any arguments, in which case it will, by default, use OpenAI model `"gpt-3.5-turbo"` with a temperature setting of 0.
+`"gpt-3.5-turbo"` points to the latest model offering enhanced speed and accuracy in generating responses in requested formats, while maintaining cost-effective performance.
+
+
+## Choosing OpenAI model
+
+When choosing OpenAI model, there are few things to keep in mind; the complexity of the task, cost associated with it and the desired response time.
```

Comment:
I would add the FAQ section before here, with most of the content written there. PLease re-read to see if everything fits. Then the read know which string to but.

Then I really like you explanation about the different series, so the reader finds out more about what he might choose.

Finally, I would delete the FAQ sections alltogether - @stichbury might have a few on where to put the authors